packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -1942,9 +1942,19 @@ describe('plugin-meetings', () => {
             });
           });
 
-          it('doesn\'t have layoutType which exists in the list of allowed layoutTypes should throw an error', async () => {
+          it('doesn\'t have layoutType which exists in the list of allowed layoutTypes should not assign for payload', async () => {
```

Comment:
I think we should leave that test as it was as we don't want to just silently ignore "invalid layout" and instead add a new test that sets the layout to some value and then calls changeVideoLayout with layoutType being undefined and verify that we send the previous value to the server.

Reply:
might be also worth testing that the scenario where changeVideoLayout is called first time on a meeting object without layoutType - in that case we should use undefined value for it as there is no previous value available

Reply:
but wouldn't it be useful to have a test that checks the following:
if some non-undefined, valid layout is set and changeLayout() is called without a layout type, then verify that we send a request without layout being specified

Reply:
I believe I understand now. I've made some changes, please re-review.